.github/workflows/dashboards-observability-test-and-build-workflow.yml

```diff
@@ -51,11 +53,20 @@ jobs:
         with:
           path: OpenSearch-Dashboards/plugins/dashboards-observability
 
+      - name: Load Plugin Bootstrap cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/node_modules
+            **/target
```

Comment:
why is it caching target?

Reply:
It showed up when I scanned for what files bootstrap was making, I assumed it may be important. I don't know the technical details of what bootstrap does, so I might have overselected.

Reply:
i'm not sure why bootstrap would touch target. target is the transpiled javascript from source code (public ts/tsx) to be executed in browser, `yarn start` (optimizing phase) and `yarn build` would update target.

caching it probably won't cause an issue even if your key does not consider source code changes (because `yarn build` will update target anyways), but i don't think it should be cached. it's not really related to dependencies